libjava/lib/src/main/java/com/mapbox/services/distance/v1/gson/DistanceGeometryDeserializer.java

```diff
@@ -0,0 +1,35 @@
+package com.mapbox.services.distance.v1.gson;
+
+import com.google.gson.JsonDeserializationContext;
+import com.google.gson.JsonDeserializer;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonParseException;
+import com.mapbox.services.commons.geojson.Geometry;
+import com.mapbox.services.commons.geojson.MultiPoint;
+
+import java.lang.reflect.Type;
+
+/**
+ * A custom deserializer that assumes a {@link MultiPoint} string.
+ *
+ * @since 2.0.0
+ */
+public class DistanceGeometryDeserializer implements JsonDeserializer<Geometry> {
+
+  /**
+   * A custom deserializer that assumes a {@link MultiPoint} string.
+   *
+   * @param json    The Json data being deserialized.
+   * @param typeOfT The type of the Object to deserialize to.
+   * @param context Context for deserialization.
+   * @return Deserialized Geometry.
+   * @throws JsonParseException This exception is raised if there is a serious issue that occurs
+   *                            during parsing of a Json string.
+   * @since 2.0.0
+   */
+  @Override
+  public Geometry deserialize(JsonElement json, Type typeOfT, JsonDeserializationContext context)
+    throws JsonParseException {
```

Comment:
it's the new style, line 31 is over the max line length so I broke it up by adding `throws` on the line below.